src/core/client.ts

```diff
@@ -238,30 +242,36 @@ class ApiClient<R extends RawApi> {
             this.hasUsedWebhookReply = true;
             const config = createJsonPayload({ ...payload, method });
             await this.webhookReplyEnvelope.send(config.body);
-            return { ok: true, result: true };
+            return { ok: true, result: true as any };
```

Comment:
why as any?

Reply:
Because for some reason the manual type annotations of the lambda function are causing TypeScript to check the result differently now, and true isn't necessarily the result that all methods `M` expect.

To be fair, I was rather surprised that the old version did not require an explicit type cast.

Reply:
Please check out #93.